src/index.html

```diff
@@ -0,0 +1,299 @@
+      <a href="./pages/pricing.html" class="header__nav-link">Pricing</a>
+      <a href="./pages/about_us.html" class="header__nav-link">About us</a>
+      <a href="./pages/blog.html" class="header__nav-link">Blog</a>
+      <a href="./pages/contact.html" class="header__nav-link">Contact</a>
+      <button type="button" class="header__button">Free trial</button>
+    </nav>
+  </header>
+
+  <section class="hero-section">
+    <div class="hero-section__text">
+      <h1 class="hero-section__heading">The Best Software to Grow your Sales and Services
+      </h1>
+      <p class="hero-section__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+        eirmod tempor
+        invidunt ut labore et
+        dolore
+        magna aliquyam erat.</p>
+    </div>
+    <form class="hero-section__form">
+      <input type="text" placeholder="Enter Your Email" class="hero-section__form__input">
+      <button type="submit" class="hero-section__button">Get free
+        trial</button>
+    </form>
+    <img class="hero-section__image" src="./assets/images/Dashboard image.png" alt="dashboard">
+  </section>
+
+  <section class="features-section">
+    <div class="features-section__text features-section__text--centered">
+      <h4 class="features-section__label">
+        High-quality</h4>
+      <h2 class="features-section__heading features-section__heading--big">We have the Best Solution for your Business
+      </h2>
+    </div>
+    <div class="features-section__cards-container">
+      <div class="features-section__card">
+        <img class="features-section__title-icon" src="./assets/icons/protected.svg" alt="security icon">
+        <div class="features-section__text">
+          <h3 class="features-section__heading">High security to protect from piracy</h3>
+          <p class="features-section__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.</p>
+        </div>
+      </div>
+      <div class="features-section__card">
+        <img class="features-section__title-icon" src="./assets/icons/thumb.svg" alt="thumbs up icon">
+        <div class="features-section__text">
+          <h3 class="features-section__heading">Premium quality performance</h3>
+          <p class="features-section__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.</p>
+        </div>
+      </div>
+      <div class="features-section__card">
+        <img class="features-section__title-icon" src="./assets/icons/headsets.svg" alt="security icon">
+        <div class="features-section__text">
+          <h3 class="features-section__heading">Full time customer support - 24/7</h3>
+          <p class="features-section__description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="why-us-section">
```

Comment:
why-us-section is not a good name, it could be named like work or business, just find a good name